results.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+import re
+
+# Part 1
+# The first part checks how many vowels there are in each
+# string. Therefore, it checks every letter and when it finds
+# a vowels it increments the counter. 
+# The second property is found using a regex, which checks
+# if there is at least a pair of the same letter. The last property
+# is found by simply checking if there are the unwanted srings.
+
+#fone = open("output_1", "w")
+
+vowels = ["a", "e", "i", "o", "u"]
+
+firstCounter = 0
```

Comment:
Small note here,[ Python variable naming convention](https://www.python.org/dev/peps/pep-0008/#naming-conventions) says that you should prefer lower case and underscored variables. I'm not a purist and I would also accept mixed case. but as a future reference, try to stick to one style: either mixed case or underscore.

Same thing applies to function naming.